pkg/controller/windowsmachine/metrics/metrics.go

```diff
@@ -95,7 +110,6 @@ func Add(ctx context.Context, cfg *rest.Config, namespace string) error {
 		if err == metrics.ErrServiceMonitorNotPresent {
 			metricsEnabled = false
 			return errors.Wrap(err, "install prometheus-operator in your cluster to create ServiceMonitor objects")
-
```

Comment:
This change can be reverted

Reply:
Do you mean delete the newline? I don't see one when I view the file https://github.com/openshift/windows-machine-config-operator/blob/8d2b8cfea355e4de75e408890a8429c917972798/pkg/controller/windowsmachine/metrics/metrics.go#L113

Reply:
or remove the check for ErrServiceMonitorNotPresent and setting metricsEnabled = false?

Reply:
A line is removed. It can be added back.